### PR TITLE
Composite Schemas Announcement blog post

### PR DESCRIPTION
**Pull request by bignimbus**
See also: https://github.com/graphql/composite-schemas-wg/discussions/157

### CODE REVIEW

File: src/pages/blog/2024-04-29-composite-schemas-announcement [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Adopting this style of collaboration has become a standard way of creating API platforms, with wide support from an array of vendors. Common patterns and best practices have been established around the various implementations and the underlying architecture has proven effective at scale. Today there are many approaches to collaborative GraphQL schema design, requiring different ways of defining the underlying schemas and composing the resulting architecture; for example Federation and Fusion take a … approach, whereas Mesh and Hasura take a … approach. API architects are having to make hard decisions early on in their projects, deciding which of the many patterns to follow.
```
Comment (bignimbus) [inline src/pages/blog/2024-04-29-composite-schemas-announcement]: This paragraph still needs content for `...` - there's a suggestion in the Google Doc that I'll run with

File: src/pages/blog/2024-04-29-composite-schemas-announcement [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Announcing the Composite Schemas Working Group"
+tags: ["announcements"]
+date: 2024-04-29
+byline: GraphQL Working Group
```
Comment (bignimbus) [inline src/pages/blog/2024-04-29-composite-schemas-announcement]: Is this the correct front matter / filename?

Comment (dimaMachina) [reply src/pages/blog/2024-04-29-composite-schemas-announcement]: file extension should be `.mdx`

File: src/pages/blog/2024-04-29-composite-schemas-announcement.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Announcing the Composite Schemas Working Group"
+tags: ["announcements"]
+date: 2024-04-29
+byline: Jeff Auriemma, Benjie Gillam
```
Comment (michaelstaib) [inline src/pages/blog/2024-04-29-composite-schemas-announcement.mdx]:
```suggestion
byline: Jeff Auriemma, Benjie Gillam, Michael Staib
```

File: src/pages/blog/2024-04-29-composite-schemas-announcement.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Announcing the Composite Schemas Working Group"
+tags: ["announcements"]
+date: 2024-04-29
+byline: Jeff Auriemma, Benjie Gillam, Michael Staib
```
Comment (kamilkisiela) [inline src/pages/blog/2024-04-29-composite-schemas-announcement.mdx]:
```suggestion
byline: Jeff Auriemma, Benjie Gillam, Michael Staib, Kamil Kisiela
```

File: src/pages/blog/2024-04-29-composite-schemas-announcement.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+Adopting this style of collaboration has become a standard way of creating API platforms, with wide support from an array of vendors. Common patterns and best practices have been established around the various implementations and the underlying architecture has proven effective at scale. Today there are many approaches to collaborative GraphQL schema design, requiring different ways of defining the underlying schemas and composing the resulting architecture; for example Federation and Fusion take an approach that optimizes for collaborative schema composition, whereas Mesh and Hasura prioritize flexibility with a variety of heterogenous services or even databases. API architects are having to make hard decisions early on in their projects, deciding which of the many patterns to follow.
```
Comment (benjie) [inline src/pages/blog/2024-04-29-composite-schemas-announcement.mdx]: This needs approval from Hasura, and I would also recommend replacing `Hasura` with the name of the specific technology we're discussing.

**Review by benjie — COMMENTED** [review]
This looks great to me; just needs Hasura sign-off :+1:

**Comment by jorydotcom** [comment]
@benjie or @dimaMachina - the Vercel check isn't passing, but is this OK to merge?

File: src/pages/blog/2024-04-29-composite-schemas-announcement.mdx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Announcing the Composite Schemas Working Group"
+tags: ["announcements"]
+date: 2024-04-29
```
Comment (benjie) [inline src/pages/blog/2024-04-29-composite-schemas-announcement.mdx]:
```suggestion
date: 2024-05-16
```

**Review by benjie — APPROVED** [review]

**Pull request merged**